api/v1beta1/openstackconfiggenerator_types.go

```diff
@@ -24,6 +24,10 @@ import (
 type OpenStackConfigGeneratorSpec struct {
 	// Name of the image used to generate configs. If missing will be set to the configured OPENSTACKCLIENT_IMAGE_URL_DEFAULT in the CSV for the OSP Director Operator.
 	ImageURL string `json:"imageURL,omitempty"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default={}
```

Comment:
well I am not sure if that really works as I was expecting it to do. we do not see a default in the rendered crd yaml. maybe we should default it in the webhook if needed.

Reply:
Does the right thing AFAICT. I deployed with and without this set

Reply:
No default in the crd yaml when it's optional maybe?